### PR TITLE
Create device for User - EXTRA: set boolean for Admin in User schema

### PR DESCRIPTION
**Pull request by Winkeltje84**
Created devise for user with extra: Admin boolean - standard to false

### CODE REVIEW

File: spec/factories/users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```
Comment (arnoFleming) [inline spec/factories/users.rb]: If you're not using it, remove it!

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```
Comment (arnoFleming) [inline spec/models/user_spec.rb]: idem

File: db/migrate/20170327131238_devise_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+class DeviseCreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+      # created extra boolean whether user is admin or not
+      t.boolean :admin,             default: false
```
Comment (arnoFleming) [inline db/migrate/20170327131238_devise_create_users.rb]: Please create test to expose/ensure that a user won't be an admin by default (as it is important business logic). Also, add a test to show your colleagues how to add an admin user to the system.

**Comment by Winkeltje84** [comment]
@arnoFleming , it should be better now.

**Comment by Winkeltje84** [comment]
please label me @leefreemanxyz !

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -6,11 +6,12 @@
     let(:guest_user) { create :user, email: "guest@user.com" }
     let(:admin_user) { create :user, email: "guest@user.com", admin: true}
 
-    it "the 'Admin' status is false" do
+    it "by standard, the 'Admin' status is false" do
       expect(guest_user.admin).to eq(false)
     end
 
-    it "the 'Admin' status is false" do
+    it "if 'Admin' status is set to true by another Admin,
```
Comment (leefreemanxyz) [inline spec/models/user_spec.rb]: I think we could describe this test as : 'it "creates an admin user when admin: true is included in the create function" '

Comment (Winkeltje84) [inline spec/models/user_spec.rb]: @lee, changed it according to your suggestion

File: spec/factories/users.rb (hunk 1 above)
Comment (Winkeltje84) [inline spec/factories/users.rb]: using it now and added factory for user

File: db/migrate/20170327131238_devise_create_users.rb (hunk 3 above)
Comment (Winkeltje84) [inline db/migrate/20170327131238_devise_create_users.rb]: added specs

**Pull request merged**